ui/appui-react/src/appui-react/frontstage/FrontstageDef.tsx

```diff
@@ -672,7 +671,7 @@ export class FrontstageDef {
   public async initializeFromProps(props: FrontstageProps): Promise<void> {
     this._id = props.id;
     this._initialProps = props;
-    this._defaultTool = props.defaultTool;
+    this._defaultTool = props.defaultTool as FrontstageProps["defaultTool"] | undefined; // Might be `undefined`, see `toFrontstageProps`.
```

Comment:
Do you really need this ?

Reply:
I think so.. Ideally we would define a separate type for this case, but should be safe enough until 4.0.